### PR TITLE
Upgrade to opam 2

### PR DESCRIPTION
**Pull request by NathanReb**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
 script: bash -ex .travis-opam.sh
 env:
-  - OCAML_VERSION=4.02 PACKAGE=enumerators
-  - OCAML_VERSION=4.03 PACKAGE=enumerators
-  - OCAML_VERSION=4.04 PACKAGE=enumerators
-os:
-  - linux
+  global:
+    - PACKAGE=enumerators
+    - DISTRO=apline
```
Comment (bbc2) [inline .travis.yml]: I don't know this distribution :)

Comment (NathanReb) [reply .travis.yml]: https://alpinelinux.org/

We've been using it in other packages, we can switch to debian if you'd prefer.

Comment (bbc2) [reply .travis.yml]: I love Alpine!  Check the spelling.  The CI passed nonetheless, which sounds fishy.

Comment (NathanReb) [reply .travis.yml]: uh! Indeed!

**Comment by NathanReb** [comment]
Ok found out what was wrong, it was not using the right script, I forgot to update it!

**Comment by NathanReb** [comment]
Hmm wait, something's weird

**Comment by NathanReb** [comment]
Ok fixed, changed the script invocation so we would see the docker buid steps as we do for other packages. Does it look good to you ?

**Comment by bbc2** [comment]
Yes, thanks!

**Pull request merged**